docs/tulu3.md

```diff
@@ -0,0 +1,360 @@
+# Tulu3 Reproduction
+
+This document details the commands and configs to reproduce the tulu3 models.
+
+## Finetuning
+
+
+### Llama-3.1-Tulu-3-8B-SFT Reproduction
+
+Below is (almost) the exact command which produced [Llama-3.1-Tulu-3-8B-SFT](https://huggingface.co/allenai/Llama-3.1-Tulu-3-8B-SFT). In our setup we ran this command on 8 machines with 64 gpus in total. 
```

Comment:
specify which type of machines?

Reply:
modified it to `We deployed the command across 8 machines, each equipped with 8 NVIDIA H100 GPUs, for a total of 64 GPUs in the our setup.`